tests/testthat/test-print.R

```diff
@@ -118,16 +45,18 @@ testthat::test_that("as_listing produces correct output when default_formatting
   )
 
   testthat::expect_error(
-    {
-      as_listing(
-        anl,
-        key_cols = "USUBJID",
-        default_formatting = list(numeric = fmt_config(align = "left"))
-      )
-    },
-    paste("Format configurations must be supplied for all listing columns.",
-          "To cover all remaining columns please add an 'all' configuration to `default_formatting`.")
-  )
+    res <- as_listing(
+      anl,
+      key_cols = "USUBJID",
+      default_formatting = list(numeric = fmt_config(align = "left"))
+    ),
+    regexp = paste0(
+      "Format configurations must be supplied for all listing columns. ",
+      "To cover all remaining columns please add an ",
+      "'all' configuration to `default_formatting`."
+    )
+  ) %>%
+    suppressMessages()
```

Comment:
messages are no more hidden. Do we need the automatic sorting message? @shajoezhu I think yes but I was wondering if there was another option for this